README.md

```diff
@@ -9,3 +9,20 @@ User can switch the open type in KUAL KPVBooklet extentions.
 
 KPVBooklet is licensed under the MIT license. See the file
 LICENSE for more details.
+
+## building
+
+- gather and install dependencies
+  - getting kindle device jars
```

Comment:
What is that for exactly?

Reply:
They're dependencies for the app. The ant script won't run without them present.

Reply:
And these Kindle device jars are proprietary I assume? That would probably preclude CI.

Reply:
Yes they are, as well as potentially FW & model-dependent in some circumstances.

Reply:
In some circumstances (not familiar with KPV, don't know if it's affected), some of the private APIs from these JARs we were relying on have been obfuscated in newer FW, which makes supporting them much harder.